### PR TITLE
Scaffolding for getRUN view

### PR DESCRIPTION
**Pull request by samsiegart**
refs: #64

Puts a placeholder view in place if the feature flag is enabled:
image

Also made some layout changes to the app to allow greater flexibility when laying out the view, and added a background gradient to spruce things up.

### CODE REVIEW

File: ui/public/index.html [hunk 1]
```diff
@@ -37,7 +37,13 @@
       href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
       rel="stylesheet"
     />
-
+    <style>
+      html {
+        height: 100vh;
```
Comment (dckc) [inline ui/public/index.html:42]: TIL:
> `height: 100vh;` means the height of this element is equal to 100% of the viewport height. -- https://stackoverflow.com/a/52941440/7963

File: ui/public/index.html [hunk 2]
```diff
@@ -37,7 +37,13 @@
...
+        background-image: linear-gradient(to top, #fff 8%, #8E23351e 70%, #fff 100%);
```
Comment (dckc) [inline ui/public/index.html:43]: I presume these colors are consistent with our branding guidelines.

File: ui/src/components/NavDrawer.js [hunk 3]
```diff
@@ -84,6 +86,9 @@ function ListItemLink(props) {
 
 function NavDrawer() {
   const classes = useStyles();
+  const {
+    state: { useRloc },
```
Comment (dckc) [inline ui/src/components/NavDrawer.js:90]:
```suggestion
    state: { useGetRUN },
```

Better to change this now than to take on tech debt.

File: ui/src/components/NavDrawer.js [hunk 4]
```diff
@@ -112,6 +117,14 @@ function NavDrawer() {
           to="/treasury"
           replace
         />
+        {useRloc && (
```
Comment (dckc) [inline ui/src/components/NavDrawer.js:120]:
```suggestion
        {useGetRUN && (
```

File: ui/src/components/Swap.jsx [hunk 5]
```diff
@@ -240,14 +244,22 @@ export default function Swap() {
 
   if (!approved) {
     return (
-      <Paper className={classes.paper}>
-        <div>To continue, please approve the Treasury Dapp in your wallet.</div>
-      </Paper>
+      <div className={classes.root}>
+        <Paper className={classes.paper}>
+          <div>
+            To continue, please approve the VaultFactory Dapp in your wallet.
```
Comment (dckc) [inline ui/src/components/Swap.jsx:250]: :+1:

**Review by dckc — APPROVED** [review]
Looks like a good start. Thanks for breaking down the commits.

Now seems like the right time to change `useRloc` to `useGetRUN`.
But I suppose it's not critical, so I leave it up to you.

File: ui/public/index.html (hunk 2 above)
Comment (samsiegart) [reply ui/public/index.html:43]: Hopefully... it's a semi-transparent shade of "merlot" from https://agoric.com/brand/. Any solid colors would be too jarring though imo. This is probably the boldest style change I'll make without product/UX input.

File: ui/src/components/NavDrawer.js (hunk 3 above)
Comment (samsiegart) [reply ui/src/components/NavDrawer.js]: Updated here and everywhere. I changed the url param to just "gr=true" for convenience though.

**Pull request merged**